mscore/scoreview.h

```diff
@@ -129,6 +129,17 @@ enum class ViewState {
       FOTO_LASSO,
       };
 
+//---------------------------------------------------------
+//   ScoreViewState
+//---------------------------------------------------------
+
+struct ScoreViewState {
+      qreal logicalZoomLevel;
```

Comment:
it's a bit safer to pre-initialize values here

Reply:
Fixed!